### DISCUSSION TITLE
Potential loot generation issue

### DISCUSSION THREAD

**Discussion by DrakiaXYZ** [sp-tarkov/server#1000] [General]
Not confirmed, opening a discussion thread so I don't lose track of it. Would need to validate the report before digging further into it.

User looted a wood crate on Woods, had overlapping items:
![image](https://github.com/user-attachments/assets/a735a3bc-63e4-4e8b-b569-456146325f05)

**Comment by Coremanite** [comment]
i am user, here is log
[LogOutput.log](https://github.com/user-attachments/files/18264142/LogOutput.log)
[2024.12.27_21-26-45_0.15.5.1.33420 traces.log](https://github.com/user-attachments/files/18264143/2024.12.27_21-26-45_0.15.5.1.33420.traces.log)
[server-2024-12-27.log](https://github.com/user-attachments/files/18264144/server-2024-12-27.log)
i go sleep now

**Comment by Drelok** [comment]
Just had it happen on customs, 
![image](https://github.com/user-attachments/assets/8695b534-16da-495e-ad15-92f38048bcf2)
